src/AddIns/Uno.UI.Lottie/Uno.UI.Lottie.Wasm.csproj

```diff
@@ -26,6 +27,7 @@
 	<ItemGroup>
 		<PackageReference Include="Uno.MonoAnalyzers" Version="1.0.0" />
 		<PackageReference Include="Uno.SourceGenerationTasks" />
+		<PackageReference Include="Newtonsoft.Json" />
```

Comment:
I'm currently trying to remove this dependency before removing the "draft" on this PR.